src/hadoop-data-node/deploy/hadoop-data-node-delete/delete-data.sh

```diff
@@ -20,11 +20,9 @@
 
 echo "Clean the hadoop-data-node's data on the disk"
 
-if [ -d "/mnt/hdfs/data" ]; then
 
-    rm -rf /mnt/hdfs/data
+rm -rf /mnt/hdfs/data*/* 2>/dev/null
```

Comment:
indent?

Reply:
It's no longer in a if block, so I removed it.